tests/test_basic/test_parse_setup_cfg.yml

```diff
@@ -1,7 +1,7 @@
 aiida_version: ~=2.0
 entry_points:
   aiida.calculations:
-  - plugin.add = aiida.calculations.arithmetic.add:ArithmeticAddCalculation
+    plugin.add: aiida.calculations.arithmetic.add:ArithmeticAddCalculation
```

Comment:
Just checking: this comes from your test infrastructure now also partially parsing the entry point string itself, right?
I.e. this is expected (?)

Reply:
Yes; it is only in setuptools that it is supplied in the string format (with the `=` delimiter) whereas pep 621, flit and poetry already have it as a dictionary